en/docs/reference/connectors/servicenow-connector/servicenow-connector-example.md

```diff
@@ -16,21 +16,78 @@ It will have two HTTP API resources, which are `postRecord` and `readRecord`.
+1. First let's create the `PostRecord` and `ReadRecord` sequences. Select Micro Integrator and click on `+` in **Sequences** to create the sequences.
 
-    <a href="{{base_path}}/assets/img/integrate/connectors/add-sequence.jpg"><img src="{{base_path}}/assets/img/integrate/connectors/add-sequence.jpg" title="Adding a Sequence" width="800" alt="Adding a Sequence"/></a>
+    <a href="{{base_path}}/assets/img/integrate/connectors/servicenow/add-sequence.png"><img src="{{base_path}}/assets/img/integrate/connectors/servicenow/add-sequence.png" title="Adding a Sequence" width="800" alt="Adding a Sequence"/></a>
+
+2. Provide the Sequence name as `PostRecord` and click on **Create**. It will open a graphical view of the sequence where you can add the mediators and connectors.  
+
+    <img src="{{base_path}}/assets/img/integrate/connectors/servicenow/add-to-sequence.png" title="Adding artifacts to Sequence" width="800" alt="Adding artifacts to Sequence"/>  
+
+3. In the graphical view click on `+` and go to the **Connectors** tab. Click on the `servicenow` connector and select `init` operation.  
+
+    <img src="{{base_path}}/assets/img/integrate/connectors/servicenow/add-servicenow-init.png" title="Adding init operation" width="800" alt="Adding init operation"/>
+
+4. Click on `+ Add Parameter` and provide the following key-value pairs. You need to use the ServiceNow instance URL, username, and password that you obtained when setting up the ServiceNow instance.
+
+    | Key | Value |
+    | --- | --- |
+    | serviceNowInstanceURL | https://dev55707.service-now.com |
+    | username | admin |
+    | password | Diazo123@ |
+   
+    Click on **Submit** to save the configuration.  
+
+5. Now add the `postRecord` operation to the sequence. Click on `+` and go to the **Connectors** tab. Click on the `servicenow` connector and select `postRecord` operation. 
+
+    <img src="{{base_path}}/assets/img/integrate/connectors/servicenow/add-servicenow-postRecord.png" title="Adding postRecord operation" width="800" alt="Adding postRecord operation"/>  
+
+6. Click on `+ Add Parameter` and provide the following key-value pairs.  
+
+    | Key | Value |
+    | --- | --- |
+    | tableName | incident |
+    | sysparmDisplayValue | true |
+    | sysparmFields | short_description,number,sys_id |
+    | sysparmView | short_description,number,sys_id |
+    | sysparmInputDisplayValue | true |
+    | number | 34 |
+    | shortDescription | {$ctx:shortDescription} |
+    | active | true |
+    | approval | owner |
+    | category | inquiry |
+    | contactType | {$ctx:contactType} |
+
+    Click on **Submit** to save the configuration.
+
+7. Add a `property` mediator to the sequence to save the `sys_id` of the created incident. Click on `+` and go to the **Mediators** tab. Click on the `property` mediator and provide the following details.  
```

Comment:
For property mediator link to the property mediator content without using backtick